lib/view/home_page.dart

```diff
@@ -61,6 +61,7 @@ class _HomePageState extends State<HomePage> {
       builder: (context, snapshot) {
         if (snapshot.hasData) {
           allLabels = _getAllLabels(of: snapshot.data!);
+          _removeSelectedDeletedLabels();
```

Comment:
I would just put the contents of this method here, because it is only 2 lines and there is no need to add another function that is only called once.